### PR TITLE
add logic to generate random nominators

### PR DESCRIPTION
**Pull request by pepoviola**
Add new `relaychain` config `random_nominators_count`, to generate random nominators and add those into the chain-spec genesis.

cc @ordian

### CODE REVIEW

File: src/chain-spec.ts [hunk 1]
```diff
@@ -433,6 +467,19 @@ export function writeChainSpec(specPath: string, chainSpec: any) {
   }
 }
 
+// helper
+function getRandom(arr: string[], n: number) {
```
Comment (wirednkod) [inline src/chain-spec.ts:475]: I think it is better for helpers to exist under `utils`?

File: src/configGenerator.ts [hunk 2]
```diff
@@ -72,6 +72,7 @@ export async function generateNetworkSpec(
       defaultImage: config.relaychain.default_image || DEFAULT_IMAGE,
       defaultCommand: config.relaychain.default_command || DEFAULT_COMMAND,
       defaultArgs: config.relaychain.default_args || [],
+      randomNominatorsCount: config.relaychain.random_nominators_count || 0,
```
Comment (wirednkod) [inline src/configGenerator.ts:75]:
```suggestion
      randomNominatorsCount: config.relaychain?.random_nominators_count || 0,
```

File: src/keys.ts [hunk 3]
```diff
@@ -9,6 +9,14 @@ function nameCase(string: string) {
   return string.charAt(0).toUpperCase() + string.slice(1);
 }
 
+export async function generateKeyFromSeed(seed: string): Promise<any> {
+  await cryptoWaitReady();
+
+  const sr_keyring = new Keyring({ type: "sr25519" });
+  const sr_account = sr_keyring.createFromUri(`//${seed}`);
+  return sr_account;
```
Comment (wirednkod) [inline src/keys.ts:17]:
```suggestion
  return sr_keyring.createFromUri(`//${seed}`);
```

File: src/chain-spec.ts (hunk 1 above)
Comment (pepoviola) [reply src/chain-spec.ts:475]: Yes, we can move to `utils` since is very generic.

Comment (pepoviola) [reply src/chain-spec.ts]: done, thanks!!

**Review by wirednkod — APPROVED** [review]

**Pull request merged**